lisbon-public-law-editions-author-date.csl

```diff
@@ -0,0 +1,591 @@
+            </group>
+          </else>
+        </choose>
+      </group>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" et-al-min="11" et-al-use-first="7" entry-spacing="0">
+    <sort>
+      <key macro="contributors-sort"/>
+      <key variable="title"/>
+      <key variable="genre"/>
+      <key variable="issued"/>
+    </sort>
+    <layout>
+      <group>
+        <group delimiter=". " suffix=".">
+          <text macro="contributors"/>
+          <text macro="title"/>
+          <text macro="in:coiso"/>
+          <choose>
+            <if type="article-journal">
+              <group delimiter=", ">
+                <text macro="locators"/>
+                <text macro="access"/>
+              </group>
+            </if>
+            <else-if type="book chapter" match="any">
+              <group delimiter=". ">
+                <choose>
+                  <if type="chapter">
+                    <text macro="container-contributors"/>
+                  </if>
+                </choose>
+                <text macro="publisher"/>
+                <text macro="access"/>
+              </group>
+            </else-if>
+            <else-if type="paper-conference">
+              <group delimiter=", ">
+                <choose>
+                  <if variable="event">
+                    <text variable="event" font-style="italic"/>
+                  </if>
+                </choose>
+                <text macro="publisher"/>
+                <text macro="locators"/>
+              </group>
+            </else-if>
+            <else-if type="patent thesis" match="any">
+              <text variable="number"/>
+              <text macro="issue"/>
+            </else-if>
+            <else-if type="bill legislation legal_case">
+              <group delimiter=", ">
+                <text variable="publisher"/>
+                <text variable="publisher-place"/>
+                <text variable="number"/>
+              </group>
+            </else-if>
+          </choose>
+          <choose>
+            <if type="patent thesis" match="none">
+              <text macro="issued"/>
+            </if>
+          </choose>
+          <group>
+            <choose>
+              <if type="book">
+                <choose>
+                  <if is-numeric="number-of-pages">
+                    <label variable="number-of-pages" form="short" suffix=" "/>
+                  </if>
+                </choose>
+                <text variable="number-of-pages"/>
+              </if>
+              <else>
+                <label variable="page" form="short" suffix=" "/>
+                <text variable="page"/>
+              </else>
+            </choose>
+          </group>
+        </group>
+        <choose>
+          <if type="article-journal" variable="DOI">
+            <text variable="DOI" prefix=" DOI: " suffix="."/>
+          </if>
+        </choose>
+        <group delimiter=" " prefix=" ">
```

Comment:
this might make more sense as an `else-if` to the previous condition? Presumably you don't want DOI and URL for journal articles.